modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductOptions.testcase

```diff
@@ -1817,6 +1817,136 @@ definition {
 		}
 	}
 
+	@description = "This is a test for COMMERCE-12167. On the Product Details page user can see option values with two different SKUs linked"
+	@priority = 5
+	test CanViewOptionsValueWithTwoSKUsLinked {
```

Comment:
CanSKUUpdateSuccessfullyWhenSelectingValuesFromMultipleOptions  

I think this would be a better title